src/Polly/Polly.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net472;net461;</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net472;net462;</TargetFrameworks>
```

Comment:
What was the motivating factor for the TFM bump?

Reply:
Needed for the [System.Diagnostics.DiagnosticSource](https://www.nuget.org/packages/System.Diagnostics.DiagnosticSource/#dependencies-body-tab) package.